EveripediaNetwork/ep-ui
# Network switch fix

# Unable to switch the network to the right network on EP-UI after login.

_It reworks the network notification such that, it only pops up when you hit the publish button on the create wiki page and you are not connected to the right network.

## How should this be tested?

1. Go to the create wiki page
2. fill in the necessary fields and hit the publish button
3 If you are connected to the right network, you don't get to see the network error notification, else, you will get the notification.

## Linked issues

fixes https://github.com/EveripediaNetwork/issues/issues/1197
